Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ruby:2-slim
+
+RUN apt-get update && \
+  apt-get install -y build-essential git
+
+RUN mkdir /app
+WORKDIR /app
+
+COPY Gemfile .
+COPY Gemfile.lock .
```

Comment:
this is an interesting dockerfile :). At first I thought it might not work, because it doesn't add the library's files. I see it doesn't actually need to, since we're not planning to use the image for anything besides local test-running.

One quirk is that when you do a fresh clone, `Gemfile.lock` doesn't exist and this image won't build, which I assume is counter to your goal of not needing ruby installed locally to hack on this project. I think just removing this line is sufficient.